apps/hs/qua-server/src/Foundation.hs

```diff
@@ -482,3 +487,13 @@ lastScenarioProblemId = getVal <$> rawSql query []
     query = unlines
           ["SELECT max(id) FROM scenario_problem;"
           ]
+
+
+maybeEnroll :: Email -> Maybe Text -> Handler ()
+maybeEnroll email msId =
+    case msId of
+        Nothing -> pure ()
+        Just i -> do
+             $(logDebug) $ "Enrolling new user with email " <> email <> " in scenario problem " <> i
+             runDB $ pure () -- TODO what here?
```

Comment:
> What do you think?

I don't like either of those options :D Let's leave them empty for now, and think about the different possible scenarios of user flow.